### ISSUE TITLE
Not seeing "IoT Hub trigger" when adding new Azure function

### ISSUE THREAD

**Issue by jlian** [Azure-Samples/azure-serverless-iot-button#5]
> Please provide us with the following information:
> ---------------------------------------------------------------

### This issue is for a: (mark with an `x`)
```
- [ ] bug report -> please search issues before submitting
- [ ] feature request
- [x] documentation issue or request
- [ ] regression (a behavior that used to work and stopped in a new release)
```

### Minimal steps to reproduce
> Open Visual Studio 2017, update to latest version, right click add new item on project, select Azure function, not seeing IoT Hub trigger

![image](https://user-images.githubusercontent.com/2320572/39774738-9767c4b8-52b0-11e8-96c8-07ed2c200908.png)

### Any log messages given by the failure
>

### Expected/desired behavior
> Documentation should provide exact version needed to see the IoT Hub trigger Azure function 

### OS and Version?
> Windows 10 

### Versions
> 15.7.0

![image](https://user-images.githubusercontent.com/2320572/39774801-cc37afd2-52b0-11e8-988f-dbe946f96015.png)

### Mention any other details that might be useful

> ---------------------------------------------------------------
> Thanks! We'll be in touch soon.

**Comment by jlian** [comment]
Fixed by #4

**Comment by jeffhollan** [comment]
Thanks for circling back
